cypresse2e/cypress/integration/e2eTests/post/post.spec.js

```diff
@@ -0,0 +1,75 @@
+import HelpBoard from '../../../elements/pages/helpBoard'
+import Post from '../../../elements/pages/post'
+
+describe('FightPandemics Post for unauthorized user', () => {
+
+    const helpBoard = new HelpBoard();
+    const post = new Post();
+    const shareViaModalWindowTitle = "Share via..."
+ 
+
+    context('User opens Help Board', () => {
+        beforeEach(() => {
+            helpBoard.visit();
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking on Like button', () => {        
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                cy.get(post.getLikeButtonSelector()).click();
+                cy.validateCorrectScreenIsOpen("auth/login");
+            })                                                      
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking on Comment button', () => {     
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                cy.get(post.getCommentButtonSelector()).click();
+                cy.validateCorrectScreenIsOpen("auth/login");
+            })                                          
+        });
+
+        it('Unauthorized user can share a post. Share via... modal window is displayed', () => {  
+            var postTitle;  
+            cy.get(helpBoard.getFirstPostOnHelpBoardSelector()).within(($firstPost) => { 
+                var postTitleElement = post.getPostTitle();
```

Comment:
What do you think about to move the variable initialization to the top of the code, to make it more consistent? Not mandatory... works as well like this, just "better practice".

Reply:
Refactored.